package.json

```diff
@@ -62,6 +62,7 @@
     }
   },
   "dependencies": {
+    "cross-spawn": "^3.0.1",
     "cross-spawn-async": "2.0.0",
```

Comment:
updated :)

On Thu, May 19, 2016 at 9:33 PM, Kent C. Dodds <notifications@github.com>
wrote:

> In package.json
> <https://github.com/kentcdodds/cross-env/pull/14#discussion_r63878305>:
>
> > @@ -62,6 +62,7 @@
> >      }
> >    },
> >    "dependencies": {
> > +    "cross-spawn": "^3.0.1",
> >      "cross-spawn-async": "2.0.0",
>
> Should we remove this now?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/kentcdodds/cross-env/pull/14/files/0d755a90a7de1bca72403e957eaffacaa69043c7#r63878305>
>